Codeinwp/neve-fse
# feat: add Otter onboarding to Neve FSE

### Summary

The PR adds an Onboarding wizard to Neve FSE that is similar to the one we previously added to Raft.

### Will affect visual aspect of the product

No

### Screenshots 


<img width="1680" alt="Screenshot 2024-08-20 at 12 03 48 AM" src="https://github.com/user-attachments/assets/d43b15f1-eb5b-437c-80d9-723ba3f71352">

### Test instructions


- Confirm the flow of Onboarding works smoothly without any debugging or console errors.
- Make sure templates look good in all color schemes.


Closes #139.
